config/initializers/carrierwave.rb

```diff
@@ -1,9 +1,9 @@
 CarrierWave.configure do |config|
   config.fog_credentials = {
-    :provider               => 'AWS',
-    :aws_access_key_id      => ENV['AWS_ACCESS_KEY_ID'],
-    :aws_secret_access_key  => ENV['AWS_SECRET_ACCESS_KEY']
+    :provider               => "AWS",
+    :aws_access_key_id      => ENV["AWS_ACCESS_KEY_ID"],
+    :aws_secret_access_key  => ENV["AWS_SECRET_ACCESS_KEY"]
   }
-  config.fog_directory  = ENV['S3_BUCKET']
-  config.fog_attributes = {'Cache-Control'=>"max-age=#{365.day.to_i}"}
+  config.fog_directory  = ENV["S3_BUCKET"]
+  config.fog_attributes = {"Cache-Control"=>"max-age=#{365.day.to_i}"}
```

Comment:
Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.